src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -356,16 +361,9 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 // for each success, write the new AccountId to the Contact.
                 c.AccountId = accountInserts[i].Id;
                 contactsToUpdate.add(c);
-                
-                //Creating affiliation - Functionality Change --> commenting out this code because we 
-                //don't want an affiliation automatically created from a Contact to its parent Account. 
-				//afflsToInsert.add(new Affiliation__c(Contact__c = c.Id, Account__c = accountInserts[i].Id, 
-				//				Primary__c = true, Status__c = 'Current'));
-				
                 i += 1;
             }
         }
-        ADDR_Contact_TDTM.contactsUpdatedAfterInsert = true;
         update contactsToUpdate;
         dmlWrapper.objectsToInsert.addAll(afflsToInsert);
```

Comment:
since you removed the affiliations logic in this method, why not get rid of the creation of afflsToInsert on line 353, as well as adding them to dmlWrapper on 368.